tests/cephfs/cephfs_system/cephfs_system_utils.py

```diff
@@ -372,3 +373,38 @@ def log_parser(self, client, expect_list, unexpect_list, daemon="mds"):
             )
             test_status = 1
         return test_status
+
+    def get_disk_usage(self, client, fs_name):
+        """
```

Comment:
Please see if doc_strings can be improved with args explanation and return value information.